ephios/core/forms/users.py

```diff
@@ -66,90 +105,48 @@ def __init__(self, **kwargs):
         if (group := kwargs.get("instance", None)) is not None:
             kwargs["initial"] = {
                 "users": group.user_set.all(),
-                "can_view_past_event": group.permissions.filter(
-                    codename="view_past_event"
-                ).exists(),
-                "can_add_event": group.permissions.filter(codename="add_event").exists(),
                 "publish_event_for_group": get_objects_for_group(
                     group, "publish_event_for_group", klass=Group
                 ),
-                "can_manage_user": group.permissions.filter(
-                    codename__in=[
-                        "add_userprofile",
-                        "change_userprofile",
-                        "delete_userprofile",
-                        "view_userprofile",
-                    ]
-                ).exists(),
-                "can_manage_group": group.permissions.filter(
-                    codename__in=[
-                        "add_group",
-                        "change_group",
-                        "delete_group",
-                        "view_group",
-                    ]
-                ).exists(),
                 "decide_workinghours_for_group": get_objects_for_group(
                     group, "decide_workinghours_for_group", klass=Group
                 ),
                 **kwargs.get("initial", {}),
             }
+            self.permission_target = group
         super().__init__(**kwargs)
         self.helper = FormHelper()
         self.helper.layout = Layout(
             Field("name"),
             Field("users"),
-            Field("can_manage_user"),
-            Field("can_manage_group"),
             Field("can_view_past_event"),
-            Field("decide_workinghours_for_group"),
-            Field("can_add_event"),
-            Field("publish_event_for_group", wrapper_class="publish-select"),
-            FormActions(Submit("submit", "Submit")),
+            Fieldset(
+                _("Management"),
+                Field("is_hr_group", title="This permission is included with the management role."),
+                "is_management_group",
+            ),
+            Fieldset(
+                _("Planning"),
+                Field(
+                    "is_planning_group",
+                    title="This permission is included with the management role.",
+                ),
+                Field("publish_event_for_group", wrapper_class="publish-select"),
+                "decide_workinghours_for_group",
+            ),
+            FormActions(Submit("submit", _("Save"))),
```

Comment:
I think this looks really clean rendered. You are really improving the permission settings IMO! :)